src/test/java/com/bakdata/common_kafka_streams/WordCountTest.java

```diff
@@ -45,11 +45,11 @@
 
     @Test
     void shouldAggregateSameWordStream() {
-        this.testTopology.input().add("bla")
+        this.testTopology.input("Input").add("bla")
```

Comment:
Did it not work before?

Reply:
It is necessary for now, but we probably should fix that in fluent-kafka-streams-test.
I gave the count store a name in the WordCount topology to make it more obvious that we use one. However, that changes the corresponding repartition topic. Without an explicit name, the store is named `KSTREAM-AGGREGATE-STATE-STORE-0000000003` and the topic `KSTREAM-AGGREGATE-STATE-STORE-0000000003-repartition`. Fluent filters `KSTREAM-*` topics so that they are not included in the input and output topics. With the store named `counts`, the topic is named `counts-repartiton` and therefore included in the input and output topic collection. Thus, I have to specify them explicitly.

Reply:
Isn't this obsolete with https://github.com/bakdata/fluent-kafka-streams-tests/pull/34?